src/components/overlays/AlertDialog/AlertDialog.stories.tsx

```diff
@@ -82,3 +89,28 @@ const Template = (props) => {
 
 export const Default = Template.bind({});
 Default.args = {};
+
+const ApiTemplate = (args) => {
+  const api = useAlertDialogApi();
+
+  return (
+    <Button
+      {...args}
+      onPress={() => {
+        const { promise } = api.open({
+          children: (
```

Comment:
It's probably better to use the `content` name here to be more consistent with other apis.